constants.gradle

```
@@ -1,10 +1,10 @@
 project.ext {
     ANDROID_MIN_SDK_VERSION = 16
-    ANDROID_TARGET_SDK_VERSION = 27
-    ANDROID_COMPILE_SDK_VERSION = 27
-    ANDROID_BUILD_TOOLS_VERSION = '27.0.3'
+    ANDROID_TARGET_SDK_VERSION = 28
+    ANDROID_COMPILE_SDK_VERSION = 28
+    ANDROID_BUILD_TOOLS_VERSION = '28.0.2'
     PLAY_SERVICES_VERSION = '11.0.4'
     GOOGLE_IMA_SDK_VERSION = '3.7.4'
-    APPCOMPAT_V7_VERSION = '27.0.0'
-    SUPPORT_V4_VERSION = '27.0.0'
+    APPCOMPAT_V7_VERSION = '27.1.1'
+    SUPPORT_V4_VERSION = '27.1.1'
```

Since we are upgrading Android dependencies to v28, should we update SUPPORT_V4_VERSION to v28?
```suggestion
    SUPPORT_V4_VERSION = '28.0.0'
```

---

I left this version because that's the version we left in the SDK. If I try to change it I get the following message:
```
Error: All com.android.support libraries must use the exact same version specification (mixing versions can lead to runtime crashes). Found versions 28.0.0, 27.1.1. Examples include com.android.support:animated-vector-drawable:28.0.0 and com.android.support:mediarouter-v7:27.1.1 [GradleCompatible]
```